test/e2e/clustertask/start_test.go

```diff
@@ -121,4 +132,95 @@ Waiting for logs to be available...
 				return nil
 			}})
 	})
+
+	t.Run("Start ClusterTask with --use-param-defaults and all the params having default", func(t *testing.T) {
+		res := tkn.MustSucceed(t, "clustertask", "start", "read-clustertask",
+			"-i=source="+tePipelineGitResourceName,
+			"-p=FILENAME=README.md",
+			"-w=name=shared-workspace,emptyDir=",
+			"--use-param-defaults",
+			"--showlog",
+			"true")
+		assert.Assert(t, is.Regexp("TaskRun started:.*", res.Stdout()))
+	})
+
+	t.Run("Start ClusterTask with --use-param-defaults and some of the params not having default", func(t *testing.T) {
+		tkn.RunInteractiveTests(t, &cli.Prompt{
+			CmdArgs: []string{"clustertask", "start", "read-clustertask",
+				"-i=source=" + tePipelineGitResourceName,
+				"--use-param-defaults",
+				"-w=name=shared-workspace,emptyDir=",
+				"--showlog",
+				"true"},
+			Procedure: func(c *expect.Console) error {
+
+				if _, err := c.ExpectString("Value for param `FILENAME` of type `string`?"); err != nil {
+					return err
+				}
+
+				if _, err := c.SendLine("README.md"); err != nil {
+					return err
+				}
+
+				if _, err := c.Send(string(terminal.KeyEnter)); err != nil {
+					return err
+				}
+
+				if _, err := c.ExpectEOF(); err != nil {
+					return err
+				}
+
+				c.Close()
+				return nil
+			}})
+	})
+
+	t.Run("Get list of TaskRuns from namespace  "+namespace, func(t *testing.T) {
```

Comment:
This test can be removed with the changes above.

Reply:
to maintain the consistency I followed the same ass it was there in exisiting [task e2e tests](https://github.com/tektoncd/cli/blob/master/test/e2e/task/start_test.go#L126). If you want then I can it as you suggested.

Reply:
Yeah, it's not necessary to check the additional TaskRuns or list. We'll know the TaskRun has succeeded based on the result of `wait.ForTaskRunState`.